core/src/main/java/org/stellar/anchor/dto/sep24/AssetResponse.java

```diff
@@ -59,4 +84,26 @@
     @SerializedName("max_amount")
     Long maxAmount;
   }
+
+  @Data
+  public static class Sep38Operation {
+    @SerializedName("exchangeable_assets")
+    List<String> exchangeableAssets;
+
+    @SerializedName("country_codes")
+    List<String> countryCodes;
+
+    @SerializedName("sell_delivery_methods")
+    List<DeliveryMethod> sellDeliveryMethods;
+
+    @SerializedName("buy_delivery_methods")
+    List<DeliveryMethod> buyDeliveryMethods;
+
+    @Data
+    public static class DeliveryMethod {
+      String name;
+
+      String description;
+    }
+  }
```

Comment:
I don't disagree with the structure of this class but its a bit confusing to have SEP-38-related code under a `/sep24/` directory, since SEP-38 isn't relevant to SEP-24.

Reply:
Great point! In addition to the SEP-38-related code there's also SEP-31 related code in the SendOperation:
https://github.com/stellar/java-stellar-anchor-sdk/blob/0b96af6ca4679ee936b0b888804d079529e7fcf4/core/src/main/java/org/stellar/anchor/dto/sep24/AssetResponse.java#L48-L61

I proposed to move this file out of `/sep24/` in another PR (https://github.com/stellar/java-stellar-anchor-sdk/pull/94) to avoid doing a broader refactor in the same PR I was adding a new feature. Would that solve your concerns?

Reply:
Yup, sounds good 👍